lib/locales/es.yml

```diff
@@ -1,27 +1,92 @@
 ---
 es:
   expectation_DoesTypeTest: '%{binding} hace comparaciones de strings innecesarias'
+  expectation_HasAnonymousVariable: '%{binding} %{must} utilizar una variable anónima'
+  expectation_HasArity0: '%{binding} %{must} tener cero parametros'
```

Comment:
Haha, yes, te problem is with "no debe": `no debe no tener parámetros` is quite awkward.